homeassistant/components/drop_connect/entity.py

```diff
@@ -0,0 +1,29 @@
+"""Base entity class for DROP entities."""
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import DROPDeviceDataUpdateCoordinator
+
+
+class DROPEntity(CoordinatorEntity[DROPDeviceDataUpdateCoordinator]):
+    """Representation of a DROP device entity."""
+
+    _attr_force_update = False
+    _attr_has_entity_name = True
+    _attr_should_poll = False
```

Comment:
Default for the coordinator entity is False, so can be omitted

Reply:
I'm not sure which entity you're referring to.  I removed `force_update` in 1a73c7b82e66eb58648e930983c3ecbc22cbe5cf.  The `should_poll` attribute appears to default to True in helpers/entity.py.

Reply:
CoordinatorEntity is a subclass of BaseCoordinatorEntity, which has `_attr_should_poll = False` already

Reply:
Got it.  Redundant attribute removed in a946e9cc404d17dea479f7dd4c2cf55acd79c258.